packages/hmr-plugin/src/hmr-manager.ts

```diff
@@ -1,13 +1,13 @@
 import { ApplicationRef, ComponentRef, NgModuleRef } from '@angular/core';
 import { InitialState, NgxsBootstrapper } from '@ngxs/store/internals';
-import { createNewHosts, removeNgStyles } from '@angularclass/hmr';
 
 import { HmrStorage } from './internal/hmr-storage';
 import {
   BootstrapModuleFn,
   NgxsHmrLifeCycle,
   NgxsHmrOptions,
-  NgxsHmrSnapshot
+  NgxsHmrSnapshot,
+  RemovableOldHostFn
```

Comment:
Maybe a better name could be `OldHostRemoverFn` because this describes what it does. What do you think?

Reply:
done